internal/lsp/code_actions.go

```diff
@@ -33,10 +33,10 @@ var (
 
 	// `source.formatAll`: Generic format code action.
 	// We do not register this for terraform to allow fine grained selection of actions.
-	// A user should be able to set `source.formatAll` to true, and source.formatAll.terraform to false to allow all
+	// A user should be able to set `source.formatAll` to true, and source.formatAll.tofu to false to allow all
```

Comment:
Can you please recheck this file? There are some references that might use some rework too.